frontend/src/Components/Charts/Sankey/WrapperSankey.tsx

```diff
@@ -0,0 +1,235 @@
+                        const to = `${nextKey}-${nextOption}`;
+                        newSankeyData.push([from, to, sankeyObj[from][to]]);
+                    });
+
+                });
+            }
+        });
+
+        //filter out empty entries
+        newSankeyData = newSankeyData.filter(d => d[2] !== 0);
+
+        stateUpdateWrapperUseJSON(sankeyData, newSankeyData, updateSankeyData);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [attributeOptions, hemoData]);
+
+
+
+    return (
+        <div style={{ minHeight: 800, width: "100%", padding: 10 }}>
+            <Typography className={`${styles.centerAlignment}`} variant="h6">
+                Outcome, Transfusion, and Result Distribution
+            </Typography>
+            <div style={{ paddingTop: 5 }}>
+                <DragDropContext onDragEnd={onDragEnd}>
+                    {attributeOptions.map((options, ind) => (
+                        <Droppable
+                            direction="horizontal"
+                            key={`${ind}-droppable`}
+                            droppableId={`${ind}`}>
+                            {(provided, snapshot) => (
+                                <div ref={provided.innerRef}
+                                    className={styles.root}
+                                    style={{ display: "flex", minHeight: '40px' }}
+                                    {...provided.droppableProps}
+                                >
+                                    <label className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-shrink MuiFormLabel-filled">{ind === 0 ? "Included" : "Not Included"}</label>
+                                    {options.map((item, index) => (
+                                        <Draggable
+                                            key={`${item.key}-draggable`}
+                                            draggableId={item.key}
+                                            index={index}
+                                        >
+                                            {(provided, snapshot) => (
+                                                <Chip
+                                                    label={item.text}
+                                                    clickable
+                                                    ref={provided.innerRef}
+                                                    {...provided.draggableProps}
+                                                    {...provided.dragHandleProps} />
+
+
+                                            )}
+                                        </Draggable>
+                                    ))}
+                                    {provided.placeholder}
+                                </div>
+                            )}
+                        </Droppable>
+                    ))}
+                </DragDropContext>
+            </div>
+            {sankeyData.length > 0 ?
```

Comment:
Is this why it doesn't mount? Maybe you need to control this components mounting based on whether the data is available yet?

Reply:
no not that it doesn't mount. the width and height of the div is very tiny on the first render